operations/service_deploy_smoke_utils.go

```diff
@@ -124,26 +124,11 @@ func checkTaskByCommit(username, key, commit string) error {
 		}
 		time.Sleep(10 * time.Second)
 		grip.Infof("checking for task %s (%d/30)", builds[0].Tasks[0], i+1)
-		r, err := http.NewRequest("GET", smokeUrlPrefix+smokeUiPort+"/rest/v2/tasks/"+builds[0].Tasks[0], nil)
-		if err != nil {
-			return errors.Wrap(err, "failed to make request")
-		}
-		r.Header.Add(evergreen.APIUserHeader, username)
-		r.Header.Add(evergreen.APIKeyHeader, key)
-		resp, err := client.Do(r)
-		if err != nil {
-			return errors.Wrap(err, "error getting task data")
-		}
-		defer resp.Body.Close()
-		body, err := ioutil.ReadAll(resp.Body)
-		if err != nil {
-			err = errors.Wrap(err, "error reading response body")
-			grip.Error(err)
-			return err
-		}
-		err = json.Unmarshal(body, &task)
+
+		var err error
+		task, err = checkTask(client, username, key, builds)
```

Comment:
this seems fine but I'm confused why its part of this commit

Reply:
It fixes a lint issue that shows up with the custom linter.